src/components/Main/NewSurveyPage/NewSurveyPage.component.jsx

```diff
@@ -3,6 +3,8 @@ import Form from '../Form/Form.component';
 import Table from '../Table/Table.component';
 import NewSurveyQuestionForm from '../NewSurveyQuestionForm/NewSurveyQuestionForm.component';
 import cloneDeep from 'lodash/cloneDeep';
+import Input from '../Input/Input.component';
```

Comment:
You can drop the word 'component' from file names. It's expected that if something in 'components' folder is a .jsx file, it's a component

Reply:
import Input from '../Input'
Input (folder): 
-index.js
-Input.sass
-Input.test.js

Reply:
done, refactored this